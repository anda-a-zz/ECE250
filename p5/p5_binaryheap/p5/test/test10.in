i 0
i 1
i 2
i 3
i 4
i 5
i 6
i 7
i 8
i 9
i 10
i 11
i 12
i 13
i 14
i 15
i 16
i 17
setd 0;1;1
setd 0;2;5
setd 0;4;6
setd 0;7;5
setd 0;14;6
setd 2;3;2
setd 2;5;8
setd 2;6;4
setd 3;10;6
setd 3;17;5
setd 4;7;3
setd 5;6;5
setd 5;9;4
setd 6;9;5
setd 6;10;3
setd 7;8;7
setd 7;11;1
setd 7;14;9
setd 8;11;6
setd 9;12;1
setd 9;13;4
setd 10;13;8
setd 10;17;2
setd 11;12;7
setd 11;14;9
setd 11;15;5
setd 12;13;4
setd 12;15;6
setd 12;16;5
setd 13;16;1
setd 13;17;4
setd 15;16;5
setd 16;17;9
shortest_d 0;1
shortest_d 0;2
shortest_d 0;3
shortest_d 0;4
shortest_d 0;5
shortest_d 0;6
shortest_d 0;7
shortest_d 0;8
shortest_d 0;9
shortest_d 0;10
shortest_d 0;11
shortest_d 0;12
shortest_d 0;13
shortest_d 0;14
shortest_d 0;15
shortest_d 0;16
shortest_d 0;17
print_path 0;1
print_path 0;2
print_path 0;3
print_path 0;4
print_path 0;5
print_path 0;6
print_path 0;7
print_path 0;8
print_path 0;9
print_path 0;10
print_path 0;11
print_path 0;12
print_path 0;13
print_path 0;14
print_path 0;15
print_path 0;16
print_path 0;17
shortest_d 1;2
shortest_d 1;3
shortest_d 1;4
shortest_d 1;5
shortest_d 1;6
shortest_d 1;7
shortest_d 1;8
shortest_d 1;9
shortest_d 1;10
shortest_d 1;11
shortest_d 1;12
shortest_d 1;13
shortest_d 1;14
shortest_d 1;15
shortest_d 1;16
shortest_d 1;17
print_path 1;2
print_path 1;3
print_path 1;4
print_path 1;5
print_path 1;6
print_path 1;7
print_path 1;8
print_path 1;9
print_path 1;10
print_path 1;11
print_path 1;12
print_path 1;13
print_path 1;14
print_path 1;15
print_path 1;16
print_path 1;17
shortest_d 2;3
shortest_d 2;4
shortest_d 2;5
shortest_d 2;6
shortest_d 2;7
shortest_d 2;8
shortest_d 2;9
shortest_d 2;10
shortest_d 2;11
shortest_d 2;12
shortest_d 2;13
shortest_d 2;14
shortest_d 2;15
shortest_d 2;16
shortest_d 2;17
print_path 2;3
print_path 2;4
print_path 2;5
print_path 2;6
print_path 2;7
print_path 2;8
print_path 2;9
print_path 2;10
print_path 2;11
print_path 2;12
print_path 2;13
print_path 2;14
print_path 2;15
print_path 2;16
print_path 2;17
shortest_d 3;4
shortest_d 3;5
shortest_d 3;6
shortest_d 3;7
shortest_d 3;8
shortest_d 3;9
shortest_d 3;10
shortest_d 3;11
shortest_d 3;12
shortest_d 3;13
shortest_d 3;14
shortest_d 3;15
shortest_d 3;16
shortest_d 3;17
print_path 3;4
print_path 3;5
print_path 3;6
print_path 3;7
print_path 3;8
print_path 3;9
print_path 3;10
print_path 3;11
print_path 3;12
print_path 3;13
print_path 3;14
print_path 3;15
print_path 3;16
print_path 3;17
shortest_d 4;5
shortest_d 4;6
shortest_d 4;7
shortest_d 4;8
shortest_d 4;9
shortest_d 4;10
shortest_d 4;11
shortest_d 4;12
shortest_d 4;13
shortest_d 4;14
shortest_d 4;15
shortest_d 4;16
shortest_d 4;17
print_path 4;5
print_path 4;6
print_path 4;7
print_path 4;8
print_path 4;9
print_path 4;10
print_path 4;11
print_path 4;12
print_path 4;13
print_path 4;14
print_path 4;15
print_path 4;16
print_path 4;17
shortest_d 5;6
shortest_d 5;7
shortest_d 5;8
shortest_d 5;9
shortest_d 5;10
shortest_d 5;11
shortest_d 5;12
shortest_d 5;13
shortest_d 5;14
shortest_d 5;15
shortest_d 5;16
shortest_d 5;17
print_path 5;6
print_path 5;7
print_path 5;8
print_path 5;9
print_path 5;10
print_path 5;11
print_path 5;12
print_path 5;13
print_path 5;14
print_path 5;15
print_path 5;16
print_path 5;17
shortest_d 6;7
shortest_d 6;8
shortest_d 6;9
shortest_d 6;10
shortest_d 6;11
shortest_d 6;12
shortest_d 6;13
shortest_d 6;14
shortest_d 6;15
shortest_d 6;16
shortest_d 6;17
print_path 6;7
print_path 6;8
print_path 6;9
print_path 6;10
print_path 6;11
print_path 6;12
print_path 6;13
print_path 6;14
print_path 6;15
print_path 6;16
print_path 6;17
shortest_d 7;8
shortest_d 7;9
shortest_d 7;10
shortest_d 7;11
shortest_d 7;12
shortest_d 7;13
shortest_d 7;14
shortest_d 7;15
shortest_d 7;16
shortest_d 7;17
print_path 7;8
print_path 7;9
print_path 7;10
print_path 7;11
print_path 7;12
print_path 7;13
print_path 7;14
print_path 7;15
print_path 7;16
print_path 7;17
shortest_d 8;9
shortest_d 8;10
shortest_d 8;11
shortest_d 8;12
shortest_d 8;13
shortest_d 8;14
shortest_d 8;15
shortest_d 8;16
shortest_d 8;17
print_path 8;9
print_path 8;10
print_path 8;11
print_path 8;12
print_path 8;13
print_path 8;14
print_path 8;15
print_path 8;16
print_path 8;17
shortest_d 9;10
shortest_d 9;11
shortest_d 9;12
shortest_d 9;13
shortest_d 9;14
shortest_d 9;15
shortest_d 9;16
shortest_d 9;17
print_path 9;10
print_path 9;11
print_path 9;12
print_path 9;13
print_path 9;14
print_path 9;15
print_path 9;16
print_path 9;17
shortest_d 10;11
shortest_d 10;12
shortest_d 10;13
shortest_d 10;14
shortest_d 10;15
shortest_d 10;16
shortest_d 10;17
print_path 10;11
print_path 10;12
print_path 10;13
print_path 10;14
print_path 10;15
print_path 10;16
print_path 10;17
shortest_d 11;12
shortest_d 11;13
shortest_d 11;14
shortest_d 11;15
shortest_d 11;16
shortest_d 11;17
print_path 11;12
print_path 11;13
print_path 11;14
print_path 11;15
print_path 11;16
print_path 11;17
shortest_d 12;13
shortest_d 12;14
shortest_d 12;15
shortest_d 12;16
shortest_d 12;17
print_path 12;13
print_path 12;14
print_path 12;15
print_path 12;16
print_path 12;17
shortest_d 13;14
shortest_d 13;15
shortest_d 13;16
shortest_d 13;17
print_path 13;14
print_path 13;15
print_path 13;16
print_path 13;17
shortest_d 14;15
shortest_d 14;16
shortest_d 14;17
print_path 14;15
print_path 14;16
print_path 14;17
shortest_d 15;16
shortest_d 15;17
print_path 15;16
print_path 15;17
shortest_d 16;17
print_path 16;17