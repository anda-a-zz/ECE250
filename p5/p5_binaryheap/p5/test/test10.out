success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
shortest distance 0 to 1 1
shortest distance 0 to 2 5
shortest distance 0 to 3 7
shortest distance 0 to 4 6
shortest distance 0 to 5 13
shortest distance 0 to 6 9
shortest distance 0 to 7 5
shortest distance 0 to 8 12
shortest distance 0 to 9 14
shortest distance 0 to 10 12
shortest distance 0 to 11 6
shortest distance 0 to 12 13
shortest distance 0 to 13 16
shortest distance 0 to 14 6
shortest distance 0 to 15 11
shortest distance 0 to 16 16
shortest distance 0 to 17 12
0 1
0 2
0 2 3
0 4
0 2 5
0 2 6
0 7
0 7 8
0 2 6 9
0 2 6 10
0 7 11
0 7 11 12
0 2 3 17 13
0 14
0 7 11 15
0 7 11 15 16
0 2 3 17
shortest distance 1 to 2 6
shortest distance 1 to 3 8
shortest distance 1 to 4 7
shortest distance 1 to 5 14
shortest distance 1 to 6 10
shortest distance 1 to 7 6
shortest distance 1 to 8 13
shortest distance 1 to 9 15
shortest distance 1 to 10 13
shortest distance 1 to 11 7
shortest distance 1 to 12 14
shortest distance 1 to 13 17
shortest distance 1 to 14 7
shortest distance 1 to 15 12
shortest distance 1 to 16 17
shortest distance 1 to 17 13
1 0 2
1 0 2 3
1 0 4
1 0 2 5
1 0 2 6
1 0 7
1 0 7 8
1 0 2 6 9
1 0 2 6 10
1 0 7 11
1 0 7 11 12
1 0 2 3 17 13
1 0 14
1 0 7 11 15
1 0 7 11 15 16
1 0 2 3 17
shortest distance 2 to 3 2
shortest distance 2 to 4 11
shortest distance 2 to 5 8
shortest distance 2 to 6 4
shortest distance 2 to 7 10
shortest distance 2 to 8 17
shortest distance 2 to 9 9
shortest distance 2 to 10 7
shortest distance 2 to 11 11
shortest distance 2 to 12 10
shortest distance 2 to 13 11
shortest distance 2 to 14 11
shortest distance 2 to 15 16
shortest distance 2 to 16 12
shortest distance 2 to 17 7
2 3
2 0 4
2 5
2 6
2 0 7
2 0 7 8
2 6 9
2 6 10
2 0 7 11
2 6 9 12
2 3 17 13
2 0 14
2 6 9 12 15
2 3 17 13 16
2 3 17
shortest distance 3 to 4 13
shortest distance 3 to 5 10
shortest distance 3 to 6 6
shortest distance 3 to 7 12
shortest distance 3 to 8 19
shortest distance 3 to 9 11
shortest distance 3 to 10 6
shortest distance 3 to 11 13
shortest distance 3 to 12 12
shortest distance 3 to 13 9
shortest distance 3 to 14 13
shortest distance 3 to 15 15
shortest distance 3 to 16 10
shortest distance 3 to 17 5
3 2 0 4
3 2 5
3 2 6
3 2 0 7
3 2 0 7 8
3 2 6 9
3 10
3 2 0 7 11
3 2 6 9 12
3 17 13
3 2 0 14
3 17 13 16 15
3 17 13 16
3 17
shortest distance 4 to 5 16
shortest distance 4 to 6 15
shortest distance 4 to 7 3
shortest distance 4 to 8 10
shortest distance 4 to 9 12
shortest distance 4 to 10 18
shortest distance 4 to 11 4
shortest distance 4 to 12 11
shortest distance 4 to 13 15
shortest distance 4 to 14 12
shortest distance 4 to 15 9
shortest distance 4 to 16 14
shortest distance 4 to 17 18
4 7 11 12 9 5
4 0 2 6
4 7
4 7 8
4 7 11 12 9
4 0 2 6 10
4 7 11
4 7 11 12
4 7 11 12 13
4 7 14
4 7 11 15
4 7 11 15 16
4 0 2 3 17
shortest distance 5 to 6 5
shortest distance 5 to 7 13
shortest distance 5 to 8 18
shortest distance 5 to 9 4
shortest distance 5 to 10 8
shortest distance 5 to 11 12
shortest distance 5 to 12 5
shortest distance 5 to 13 8
shortest distance 5 to 14 19
shortest distance 5 to 15 11
shortest distance 5 to 16 9
shortest distance 5 to 17 10
5 6
5 9 12 11 7
5 9 12 11 8
5 9
5 6 10
5 9 12 11
5 9 12
5 9 13
5 2 0 14
5 9 12 15
5 9 13 16
5 6 10 17
shortest distance 6 to 7 14
shortest distance 6 to 8 19
shortest distance 6 to 9 5
shortest distance 6 to 10 3
shortest distance 6 to 11 13
shortest distance 6 to 12 6
shortest distance 6 to 13 9
shortest distance 6 to 14 15
shortest distance 6 to 15 12
shortest distance 6 to 16 10
shortest distance 6 to 17 5
6 2 0 7
6 9 12 11 8
6 9
6 10
6 9 12 11
6 9 12
6 10 17 13
6 2 0 14
6 9 12 15
6 10 17 13 16
6 10 17
shortest distance 7 to 8 7
shortest distance 7 to 9 9
shortest distance 7 to 10 17
shortest distance 7 to 11 1
shortest distance 7 to 12 8
shortest distance 7 to 13 12
shortest distance 7 to 14 9
shortest distance 7 to 15 6
shortest distance 7 to 16 11
shortest distance 7 to 17 16
7 8
7 11 12 9
7 11 12 9 6 10
7 11
7 11 12
7 11 12 13
7 14
7 11 15
7 11 15 16
7 11 12 13 17
shortest distance 8 to 9 14
shortest distance 8 to 10 22
shortest distance 8 to 11 6
shortest distance 8 to 12 13
shortest distance 8 to 13 17
shortest distance 8 to 14 15
shortest distance 8 to 15 11
shortest distance 8 to 16 16
shortest distance 8 to 17 21
8 11 12 9
8 11 12 9 6 10
8 11
8 11 12
8 11 12 13
8 11 14
8 11 15
8 11 15 16
8 11 12 13 17
shortest distance 9 to 10 8
shortest distance 9 to 11 8
shortest distance 9 to 12 1
shortest distance 9 to 13 4
shortest distance 9 to 14 17
shortest distance 9 to 15 7
shortest distance 9 to 16 5
shortest distance 9 to 17 8
9 6 10
9 12 11
9 12
9 13
9 12 11 14
9 12 15
9 13 16
9 13 17
shortest distance 10 to 11 16
shortest distance 10 to 12 9
shortest distance 10 to 13 6
shortest distance 10 to 14 18
shortest distance 10 to 15 12
shortest distance 10 to 16 7
shortest distance 10 to 17 2
10 6 9 12 11
10 6 9 12
10 17 13
10 6 2 0 14
10 17 13 16 15
10 17 13 16
10 17
shortest distance 11 to 12 7
shortest distance 11 to 13 11
shortest distance 11 to 14 9
shortest distance 11 to 15 5
shortest distance 11 to 16 10
shortest distance 11 to 17 15
11 12
11 12 13
11 14
11 15
11 15 16
11 12 13 17
shortest distance 12 to 13 4
shortest distance 12 to 14 16
shortest distance 12 to 15 6
shortest distance 12 to 16 5
shortest distance 12 to 17 8
12 13
12 11 14
12 15
12 16
12 13 17
shortest distance 13 to 14 20
shortest distance 13 to 15 6
shortest distance 13 to 16 1
shortest distance 13 to 17 4
13 12 11 14
13 16 15
13 16
13 17
shortest distance 14 to 15 14
shortest distance 14 to 16 19
shortest distance 14 to 17 18
14 11 15
14 11 15 16
14 0 2 3 17
shortest distance 15 to 16 5
shortest distance 15 to 17 10
15 16
15 16 13 17
shortest distance 16 to 17 5
16 13 17